docs/user/docs/getting-started/working-with-git.md

```diff
@@ -110,13 +110,136 @@ there. Having done that you can click "Push".
+stable the option "Capella: Match equivalent technical elements" also worked in
+the past. If nothing else helps choose "Names" and click "Use labels as names"
+to match but labels easily change.
+
+![Configure Matching Strategie](working-with-git/configure-merge-matching.png)
+
+Afterwards, click "Ok" and "Finish".
+
+Then you get into the merge view. Market with the lock you can see which model
+is going to be modified. They are also labeled as source and target where the
+source is from which we are taking the changes and the target is the model that
+gets modified.
+
+![Explain Merge View](working-with-git/explain-merge-view.png)
+
+The colors highlight what change was made and will be applied to the target if
+you apply the change: _blue: create, red: delete, purple: modify_.
+
+If there are any objects that are overarching you have to merge them seperately
+as they can not be merged in the seperate layers.
+
+### Merge Model Elements
+
+In the synthesis view you can see the number of changes that happened. If you
+click on a specific change you can see in the Details (1) and also marked in
+blue on both models what changed and possibly how the current status in both
+models is. Then you can right click on the change in the synthesis view and
+click "Apply change(s)" (2) and you can click "Ok" then. Afterwards make sure
+to Ctrl+S or File > Save to save the change made.
+
+<!-- prettier-ignore -->
+!!! warning
+
+      If any of your changes depend on common objects independent of layers make sure
+      to also apply those changes in order to create a working modification of the
+      model.
+
+If you have also changes in diagrams you should migrate them as well in the
+same session.
+
+### Merge Diagrams
+
+Diagrams are stored in the `.aird` file. There the application of changes
+happens similar. You can have a look into the multiple stages to see the what
+exactly changed in the diagrams and you can very detailed choose the changes
+that you want to apply.
+
+If you have also changes in the model content you should merge both in the same
+session.
+
 ## Merge Conflicts
 
 Commit your changes as described above. Pull from the remote (e.g. right click
 on the project > "Team" > "Pull"). This might result in merge conflicts which
 you can solve via the merge tool.
 
-If this is not the case you have to open the reset view.
+### Merging with the Merge Tool
+
+In the above description we made sure to decide which model is our target and
+is going to be modified while the changes would be taken from the source model.
+In the case of a merge conflict we have changes in both models at the same
+position and it is not possible to decide which change to keep and which to
+delete while possibly both are reasonable changes. Thus we as users have to
+decide that and both changes have the same right to be present in the model. In
+order to not just apply changes you can decide from which sides changes can be
+taken.
```

Comment:
This paragraph doesn't read fluently. This is about the case where the tooling cannot automatically merge and a user has to do it by hand, right?

Reply:
I agree with Arjan that we should rewrite this paragraph. 

Especially the following two parts:

> In the case of a merge conflict we have changes in both models at the same
position and it is not possible to decide which change to keep and which to
delete while possibly both are reasonable changes.

It's possible to decide which changes to keep and which to discard, but the user has to do it. There is just no automatic way to do it.

> Thus we as users have to
decide that and both changes have the same right to be present in the model. In
order to not just apply changes you can decide from which sides changes can be
taken.

This sentence is a bit confusing in terms of structure and clarity.

Reply:
I tried to make it a bit more clear